about.qmd

```diff
@@ -35,8 +35,10 @@ Code in this repo is provided without warranty or promise of support. This is no
 
 ## Data Formats
 
-- [ ] File structure output by the AtoMx export module
-- [ ] Converting between data formats: AtoMx flat files, Seurat, squidpy
+- [X] [File structures for outputs of the AtoMx&#8482; export module](posts/segmentation-error-evaluation/assets/Pancreas-CosMx-ReadMe.html). <br/>
```

Comment:
`{target="_blank"}` added.
The readme html is first used by the FastReseg post and that's why it's in that post's asset. 
Since the readme html is now being linked to other part of the repository, I have moved it to the `assets` folder directly under the repos root folder. 

I don't have the Rmd used to generate this particular ReadMe html; besides, I don't want to add all the big flat files to the repository. I think it's ok to show html directly. 

Unfortunately, I don't know how to make html file into its own post under the "all" categories page. The best I can do after digging is to add it to the `about.qmd` page, which would allow it to show up in browse by topic page.